etc/ci/report_aggregated_expected_results.py

```diff
@@ -184,7 +184,13 @@ def get_pr_number() -> Optional[str]:
     return None
 
 
-def create_check_run(body: str, tag: str = ""):
+def create_report_on_github(body: str, tag: str = ""):
+    # create github job summery
+    if "GITHUB_STEP_SUMMARY" in os.environ:
+        with open(os.environ["GITHUB_STEP_SUMMARY"], "a") as f:
+            print(body, file=f)
+
+    # create github check run (more user friendly but very limited report size)
```

Comment:
```suggestion
    # Create a GitHub check run. This is more user friendly than a job summary,
    # but has a very limited size -- thus it may fail.
```